File: Big O Notation/Linear Complexity/Golang/LinearComplexity.go
package main

import (
	"fmt"
	"math"
)

func main() {
	inputs := [6]int{2, 3, 4, 5, 6, 7}
	display_all_cubes(inputs)
}

func display_all_cubes(items [6]int) {
	for _, item := range items {
		result := math.Pow(float64(item), float64(3))
		fmt.Println(result)
	}
}

File: Big O Notation/Linear Complexity/Typescript/LinearComplexity.ts
function display_all_cubes(items : number[]){
    items.forEach(item => {
        var result=Math.pow(item, 3);
        console.log(result);        
    });
}

let inputsLC: number[]=[2,3,4,5,6,7];
display_all_cubes(inputsLC);



